TYPO3.Neos/Classes/TYPO3/Neos/Utility/BackendAssetsUtility.php

```diff
@@ -39,7 +39,7 @@ public function injectSettings(array $settings)
      */
     public function shouldLoadMinifiedJavascript()
     {
-        return isset($this->settings['userInterface']['loadMinifiedJavaScript']) ? $this->settings['userInterface']['loadMinifiedJavaScript'] : $this->settings['userInterface']['loadMinifiedJavascript'];
+        return isset($this->settings['userInterface']['loadMinifiedJavaScript']) ? $this->settings['userInterface']['loadMinifiedJavaScript'] : true;
```

Comment:
Would also ignore configuration errors. Otherwise we would have to check for each config key.